### PR TITLE
Fix Travis CI and update README.md to show Travis results

### PR DESCRIPTION
**Pull request by jeffaco**
@Microsoft/ostc-devs 
@Microsoft/omi-devs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Open Management Infrastructure
+# Open Management Infrastructure [![Build Status](https://travis-ci.org/Microsoft/omi.svg?branch=master)](https://travis-ci.org/Microsoft/omi)
```
Comment (andschwa) [inline README.md]: :+1:

File: Unix/configure [hunk 2]
```diff
@@ -245,6 +245,11 @@ do
         enable_sections=1
         ;;
 
+    --show-version)
+        echo $version
+	exit 0
```
Comment (andschwa) [inline Unix/configure]: There's a tab here.

File: Unix/regress [hunk 3]
```diff
@@ -182,10 +182,7 @@ if [ "$run_check" = "1" ]; then
 rm -rf ./output4 GNUmakefile
 
 # compose omi tar package name
-major=`cut -d. -f1 $root/VERSION`
-minor=`cut -d. -f2 $root/VERSION`
-revision=`cut -d. -f3 $root/VERSION`
-version="$major.$minor.$revision"
+version="`$root/configure --show-version`"
```
Comment (andschwa) [inline Unix/regress]: LGTM.

**Comment by niroyb** [comment]
Signed off

**Pull request closed without merging**